libs/store_helper/kernel/ebpf_registry_helper_km.c

```diff
@@ -84,15 +73,15 @@ write_registry_value_ansi_string(ebpf_registry_key_t key, _In_z_ const wchar_t*
     return status;
 }
 
-static _Must_inspect_result_ ebpf_registry_result_t
+_Must_inspect_result_ ebpf_registry_result_t
 write_registry_value_dword(ebpf_registry_key_t key, _In_z_ const wchar_t* value_name, uint32_t value)
 {
     UNICODE_STRING unicode_name;
     RtlInitUnicodeString(&unicode_name, value_name);
     return ZwSetValueKey(key, &unicode_name, 0, REG_DWORD, &value, sizeof(uint32_t));
 }
 
-static _Must_inspect_result_ ebpf_registry_result_t
+_Must_inspect_result_ ebpf_registry_result_t
```

Comment:
I'd like these to use ebpf_result_t instead of ebpf_registry_result_t in the future.
But since they were already this way, this is not a blocking comment for this PR.

Reply:
Converted all low-level registry functions to return ebpf_result_t